src/screens/Assistants/EmployeeTable.jsx

```diff
@@ -58,22 +59,16 @@ class EmployeeTable extends React.Component {
 
   handleStatus = (event) => {
     if (event.value == "Administrator") {
-      const newRoleName = "admin";
-      this.props.users
-        .filter((user) => user == this.state.userSelectedForEdit)
-        .map((selectedUser) => (selectedUser.role = newRoleName));
```

Comment:
Can we keep this part? This updates the role in the UI table after the modal is closed; else refresh is required. (same for the case below).